### DISCUSSION TITLE
Postgres 13 and 14 compatibility?

### DISCUSSION THREAD

**Discussion by Fizzizist** [rdkit/rdkit#4574] [Q&A] [answered]
When I attempt to install the postgres cartridge for postgres versions 13 and 14, I encounter cmake errors. Just wondering if there are plans to support these newer versions in the future?

I am using Ubuntu 20.04 if that is relevant.

**Comment by ptosco** [comment, marked as answer]
@Fizzizist I have built the latest `master` against Postgres 13 without issues. What errors are you getting?

**Reply by Fizzizist** [reply]
ah perhaps I just have an outdated version. We use `Release_2020_03_4`. Here is the output I get when attempting to install:
![image](https://user-images.githubusercontent.com/16968180/135346121-4b10fe62-cc45-4f66-bdfb-bd0368114bd5.png)

**Reply by Fizzizist** [reply]
update: it seems that when I try it with `Release_2021_03_5` I actually get the same error. Do you want to see the output of `CMakeError.log`?

**Reply by ptosco** [reply]
@Fizzizist the definitions I use to point `cmake` to my PostgreSQL 13 installation are:
```
        -D PostgreSQL_INCLUDE_DIR=/scratch/toscopa1/install/postgres_13_1/include \
        -D PostgreSQL_TYPE_INCLUDE_DIR=/scratch/toscopa1/install/postgres_13_1/include/server \
        -D PostgreSQL_LIBRARY=/scratch/toscopa1/install/postgres_13_1/lib/libpq.so \
```
I do not set `PostgreSQL_ROOT` as I don't think it is honored anymore.
AFAICR I figured out the above variables looking into the `FindPostgreSQL.cmake` source.

**Reply by Fizzizist** [reply]
The above works for me. I do have to upgrade versions to get it working, but setting those 3 variables and creating a symlink to libpq.so to the correct directory does work

**Reply by ptosco** [reply]
Glad that it works!
